src/openvic-simulation/map/Map.cpp

```diff
@@ -229,15 +230,15 @@ bool Map::generate_mapmode_colours(Mapmode::index_t index, uint8_t* target) cons
 		const colour_t base_colour = static_cast<colour_t>(base_stripe);
 		const colour_t stripe_colour = static_cast<colour_t>(base_stripe >> (sizeof(colour_t) * 8));
```

Comment:
This is what I was referring to earlier, the only place where `base_stripe_t` is deconstructed. Having the component colours as member variables would be cleaner, and wouldn't break anything else so long as `sizeof(base_stripe_t) == 2 * sizeof(colour_t) == 8` still holds true.